l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1044,6 +1046,44 @@ def set_art73(self, FatturaBody, invoice_data):
         if FatturaBody.DatiGenerali.DatiGeneraliDocumento.Art73:
             invoice_data['art73'] = True
 
+    def set_roundings(self, FatturaBody, invoice):
+        rounding = 0.0
+        if FatturaBody.DatiBeniServizi.DatiRiepilogo:
+            for summary in FatturaBody.DatiBeniServizi.DatiRiepilogo:
+                rounding += float(summary.Arrotondamento or 0.0)
+        if FatturaBody.DatiGenerali.DatiGeneraliDocumento:
+            summary = FatturaBody.DatiGenerali.DatiGeneraliDocumento
+            rounding += float(summary.Arrotondamento or 0.0)
+        if rounding:
+            arrotondamenti_attivi_account_id = self.env.user.company_id.\
+                arrotondamenti_attivi_account_id
+            arrotondamenti_passivi_account_id = self.env.user.company_id.\
+                arrotondamenti_passivi_account_id
+            arrotondamenti_tax_id = self.env.user.company_id.\
+                arrotondamenti_tax_id
```

Comment:
Penso sia meglio controllare se questi campi sulla company siano vuoti, così da avvisare l'utente che deve inserirli in caso di ricezione di una fattura con arrotondamento

Reply:
@eLBati Ok! Dove pensi vadano bene? Facciamo una ``raise`` direttamente nel codice qui sopra?

Reply:
Direi di sì.
Anche perchè creando una riga senza `account_id` si otterrebbe cmq un errore bloccante, ma meno significativo.

Ho solo il dubbio su `arrotondamenti_tax_id`: se manca, la riga può essere creata lo stesso, anche se sbgliata. Controlliamo anche lui?

Reply:
@eLBati In linea generale ``arrotondamenti_tax_id`` dovrebbe essere IVA fuori campo, quindi potremmo limitarci a fare la registrazione senza IVA, registrare l'incongruenza e notificarla all'utente.

Reply:
@eLBati Ho appena pushato la modifica

Reply:
:+1: